Chapter02/recipe-0011/README.md

```diff
@@ -0,0 +1,100 @@
+# Discovering the operating system
+
+CMake is a set of cross-platform tools. Nevertheless it can be very useful to
+know on which operating system (OS) the configure or build step is executed:
+either to tweak CMake code for a particular OS, to enable conditional
+compilation depending on the OS or to use compiler-specific extensions if
+available. In this recipe we will demonstrate how to use CMake to detect the
+OS.
+
+
+## Ingredients
+
+- A C++ compiler
+- CMake (any version higher than 3.0)
+
+
+## How to do it
+
+For this example we will use the following `CMakeLists.txt` together with a
+`hello-world.cpp` that we can reuse from recipe X (ADD LINK) although the C++
+source code does not matter for the essence of this recipe and you can even
+leave it out and comment out the last line containing
```

Comment:
This is redundant, I think. By the way, I've started commenting on the source files under "Getting ready", seems more appropriate.